app/utils/pyodide/webworker.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This file has been copied from pyodide source and modified to allow
+ * pyodide to be used in a web worker within this
+ */
+
+self.languagePluginUrl = './src';
+importScripts('./pyodide.js');
```

Comment:
likely need to migrate to using the updated `pyodide.js` bundled with the updated pyodide